docs/platforms/javascript/common/profiling/index.mdx

```diff
@@ -26,15 +26,15 @@ Note that since the profiling API is currently only exposed in Chromium, profile
 
 To get started with JavaScript browser profiling, you'll need to:
 
-- Install the @sentry/browser SDK, minimum version 7.60.0
+- Install the <PlatformSdkPackageName fallback="@sentry/browser"/> SDK, minimum version 7.60.0
 - Configure the document response header to include `Document-Policy: js-profiling`
 - Configure the SDK to use the `BrowserProfilingIntegration` and set `profilesSampleRate`
 
-## Step 1: Install the JavaScript Browser SDK
+## Step 1: Install the <PlatformOrGuideName/> SDK
 
 <PlatformSection notSupported={["javascript.electron"]}>
 
-Install our JavaScript browser SDK using either `yarn` or `npm`, the minimum version that supports profiling is **7.60.0**.
+Install our <PlatformOrGuideName/> SDK using either `yarn` or `npm`, the minimum version that supports profiling is **7.60.0**.
```

Comment:
This is the concrete use case I want to fix (which was reported by our sales engineers today in Slack).